frontend/layouts/shelter/ShelterLayout/HeaderContent.tsx

```diff
@@ -1,12 +1,21 @@
 import { Menu } from "antd";
+import { MenuClickEventHandler } from "rc-menu/lib/interface";
```

Comment:
As someone recommended to satisfy type checker. 
https://github.com/ant-design/ant-design/issues/25467#issuecomment-787128441